low-level/framework/utils/utility.py

```diff
@@ -58,3 +60,55 @@ def is_env_vm(self):
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
 
+    @staticmethod
+    def send_command(command: str, fail_on_error=True):
+        # Note: This function uses subprocess to execute commands, scripts which are not possible to execute
+        # through any python routines available. So its usage MUST be limited and used only when no other
+        # alternative found.
+        output, error, returncode = SimpleProcess(command).run()
+        if returncode != 0:
+            print("command '%s' failed with error\n%s" % (command, error))
+            if fail_on_error:
+                sys.exit(1)
+            else:
+                return str(error)
+        return str(output)
+
+    @staticmethod
```

Comment:
add code comments here on why we need this function when Simpleprocess if there.

Reply:
Done